godot-core/src/registry/callbacks.rs

```diff
@@ -33,7 +33,14 @@ pub unsafe extern "C" fn create<T: cap::GodotDefault>(
     _class_userdata: *mut std::ffi::c_void,
     _notify_postinitialize: sys::GDExtensionBool,
 ) -> sys::GDExtensionObjectPtr {
-    create_custom(T::__godot_user_init).unwrap_or(std::ptr::null_mut())
+    if let Ok(object_ptr) = create_custom(T::__godot_user_init) {
+        let mut obj = Gd::<T>::from_obj_sys_weak(object_ptr).upcast_object();
+        obj.notify(crate::classes::notify::ObjectNotification::POSTINITIALIZE);
+        std::mem::forget(obj);
+        object_ptr
```

Comment:
Is it possible to directly use
```rs
Gd::<Object>::from_obj_sys_weak(object_ptr)
``` without `upcast_object()`?

Also, this creation of a weak pointer would need some comments, the context won't be clear in a year otherwise :slightly_smiling_face: